01_GrC_2020_regular/Cereb_GrC_regular.py

```diff
@@ -185,8 +185,9 @@ def __init__(self, _id):
         secnumber_pf = int(1000/lensec)
 
         self.ais.connect(self.axon,1,0)
-
-
+          
+        len_initial_ais = -16.62232
```

Comment:
we may want to look at this... negative lengths aren't usually the best way to do things